lib/assets/stylesheets/_core.scss

```diff
@@ -36,12 +57,12 @@ $desktop-breakpoint: 992px !default;
 @import "accessibility";
 
 html, body {
```

Comment:
Ideally the template should be updated to use the classes from https://github.com/alphagov/govuk-frontend/blob/master/src/core/_template.scss